charts/datahub/subcharts/datahub-mae-consumer/values.yaml

```diff
@@ -203,6 +203,9 @@ global:
     ## Enables always emitting a MCL even when no changes are detected. Used for Time Based Lineage when no changes occur.
     alwaysEmitChangeLog: true
 
+    ## Enables diff mode for graph writes, uses a different code path that produces a diff from previous to next to write relationships instead of wholesale deleting edges and reading
+    enableGraphDiffMode: true
```

Comment:
This should maybe be a global value? Don't think we would want to set this differently per service, but maybe it provides flexibility for a situation I'm not thinking of.